src/server/app.js

```diff
@@ -18,6 +18,27 @@ app.get('/api/disorder/search', (req, res) => {
   });
 });
 
+app.get('/api/getProfile/:ID', (req, res) => {
```

Comment:
I suggest using one endpoint for profile api. `api.get('/api/profile...')` and ``api.put('/api/profile...')``

Reply:
Ok. Get API is still work in progress so I will incorporate the comment in next update